### PR TITLE
Kinder: add support for testing kubeadm-kustomize

### PR DESCRIPTION
**Pull request by fabriziopandini**
This PR implements the required changes in order to use kinder for testing kubeadm-kustomize feature

/assign @neolit123

### CODE REVIEW

**Comment by neolit123** [comment]
LGTM on a quick look, will double check again once https://github.com/kubernetes/kubernetes/pull/81458 merges.

thanks.

File: kinder/ci/workflows/kustomize-master.yaml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+summary: |
+  This workflow implements a sequence of tasks used for testing alternative
+  discovery methods for kubeadm join.
+vars:
+  kubernetesVersion: "{{ resolve `ci/latest` }}"
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-kustomize
+tasks:
+#  - name: pull-base-image
```
Comment (Klaven) [inline kinder/ci/workflows/kustomize-master.yaml]: is there a reason to leave this in?

File: kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go [hunk 2]
```diff
@@ -141,8 +159,14 @@ func kubeadmUpgradeNode(c *status.Cluster, n *status.Node, upgradeVersion *K8sVe
 	// before two actions where required
 	if n.MustKubeadmVersion().AtLeast(constants.V1_15) {
 		// kubeadm upgrade node
+		nodeArgs := []string{
+			"upgrade", "node", fmt.Sprintf("--v=%d", vLevel),
+		}
+		if kustomizeDir != "" {
```
Comment (Klaven) [inline kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go]: I thought this needed to be version at least `V1_16`?

File: kinder/ci/workflows/kustomize-master.yaml (hunk 1 above)
Comment (fabriziopandini) [reply kinder/ci/workflows/kustomize-master.yaml]: fixed

File: kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go (hunk 2 above)
Comment (fabriziopandini) [reply kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go]: Yes, this is tested up in the chain when `KubeadmUpgrade` starts

**Comment by neolit123** [comment]
/lgtm
@fabriziopandini want me to send the test-infra PR?

**Pull request merged**